modules/c++/math.poly/include/math/poly/Fit.h

```diff
@@ -250,6 +251,79 @@ inline math::poly::TwoD<double> fit(size_t numRows,
 
     return fit(xm, ym, zm, nx, ny);
 }
+
+typedef math::linear::VectorN<3, double> Vector3;
+
+inline math::poly::OneD<Vector3> fit(
```

Comment:
For this and all the functions in here, use Doxygen style comments along these lines:
```
    /*!
     * Returns a scaled polynomial such that
     * P'(x, y) = P(x * scaleX, y * scaleY)
     *
     * \param scaleX The factor to apply to x
     * \param scaleY The factor to apply to y
     *
     * \return Scaled polynomial of the same order as the original polynomial
     */
```